packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -1014,6 +1026,81 @@ export const useArbTokenBridge = (
     )
   }
 
+  // mutates provided array - assigns deadlineBlockNumbers to all asserted but unconfimred l2tol1 messages 
+  const addNodeDeadlineToUnconfirmedWithdrawals = async(l2ToL1Data: L2ToL1EventResultPlus[])=>{
+    if(l2ToL1Data.length === 0) return []
+    const l1NetworkID = await l1NetworkIDCached()
+    const oldOutboxOffset = (()=>{
+      switch (l1NetworkID) {
+        case '1':
+          return 30
+        case '4':
+          return 326
+        default:
+          throw new Error(`Unrecognized L1 network ${l1NetworkID}`)
+      }
+    })()
+
+    // get smallest batch number in messages for lower bound on graph query    
+    const smallestBatchNumber = l2ToL1Data.reduce((acc: BigNumber, currentL2ToL1Data:L2ToL1EventResultPlus )=>{
+      return acc.lt(currentL2ToL1Data.batchNumber) ? acc : currentL2ToL1Data.batchNumber
+    }, l2ToL1Data[0].batchNumber).toNumber()    
+    const nodes = await getNodes(l1NetworkID, smallestBatchNumber)
+  
+
+    const unconfirmedWithdrawals = l2ToL1Data.filter((l2ToL1Data)=> l2ToL1Data.outgoingMessageState !==  OutgoingMessageState.CONFIRMED && l2ToL1Data.outgoingMessageState !==  OutgoingMessageState.EXECUTED ).sort((msgA,msgB)=>{
```

Comment:
very long line grrr

Reply:
linter will make everything okay (but even so i shortened it by 37 chars, just for you)